lean/main/syntax.lean

```diff
@@ -159,15 +159,18 @@ def Playground1.test := MyTerm
 --   MyTerm
 
 /-!
-Implementing this so called "elaboration function", which will actually
-give meaning to this syntax, is topic of the elaboration and macro chapter.
-An example of one we have already seen however would be the `notation` and
-`infix` command.
```

Comment:
It looks like these in particular are macros rather than elaborators (hence the change here).